### PR TITLE
Add option to toggle framebuffer acceleration when using software rendering.

### PR DESCRIPTION
**Pull request by Dyrewulfe**
See #20987

Added as a simple toggled option. May or may not help performance, depending on the system. Can make a significant difference with integrated graphics on Windows(2 to 5 times fps on my system).

### CODE REVIEW

**Comment by BrettDong** [comment]
Is it a 'restart to apply' option? Would be better If add the hint in the description.

File: src/sdltiles.cpp [hunk 1]
```diff
@@ -395,6 +395,8 @@ bool WinCreate()
         }
     }
     if( software_renderer ) {
+        if( get_option<bool>( "FRAMEBUFFER_ACCEL" ) )
+            SDL_SetHint( SDL_HINT_FRAMEBUFFER_ACCELERATION, "1" );
```
Comment (Coolthulhu) [inline src/sdltiles.cpp]: Coding style: even single statement `if`s/`while`s/etc. should get braces.

**Comment by narc0tiq** [comment]
**Jenkins, rebuild** this please.

(boldface not required, actual regex is `.*jenkins,?\W+rebuild.*`, case-insensitive)

File: src/sdltiles.cpp (hunk 1 above)
Comment (Graywolfe813) [inline src/sdltiles.cpp]: This isn't effectively detailed in CODE_STYLE.md. Is there a more detailed outline?

Same line braced, or K&R style for one liners like this?

**Comment by Coolthulhu** [comment]
I'm not seeing any difference in FPS on my setup.
Can someone else confirm the FPS gains?

**Pull request merged**